chapters/synchronous.tex

```diff
@@ -412,9 +412,10 @@ \section{Clock Constructors}\label{clock-constructors}
 \firstuse[event clock]{Event clock}.
 The first input argument, $\mathit{condition}$, is a continuous-time expression of type \lstinline!Boolean!.
 The optional $\mathit{startInterval}$ argument (defaults to 0) is the value returned by \lstinline!interval()! at the first tick of the clock, see \cref{initialization-of-clocked-partitions}.
-The result is of base type \lstinline!Clock! that ticks when \lstinline!edge($\mathit{condition}$)! becomes \lstinline!true!.
+The result is of base type \lstinline!Clock! that ticks when \lstinline!edge(pre($\mathit{condition}$))! becomes \lstinline!true!.
 \begin{nonnormative}
 This clock is used to trigger a clocked base-partition due to a state event (that is a zero-crossing of a \lstinline!Real! variable) in an unclocked base-partition, or due to a hardware interrupt that is modeled as \lstinline!Boolean! in the simulation model.
+The additional \lstinline!pre! delays the event clock one event iteration, but does not introduce any time-delay.
```

Comment:
If the clock is delayed one even iteration, doesn't it also mean that there is a time delay? Isn't there some time passed between one iteration and another?

Reply:
No time delay. 
It may be that we should be clearer with the terminology - and say "one step of the event iteration", even if reinit uses similar terminology. Basically we have an "event" at a point in time and "iterate" (at that time-instant), and the event clock is activated in the next iteration (at that time-instant).

So:
- Over-all procedure: currently event
- Each step in the iterative: currently event iteration, proposed "one step of the event iteration"

Reply:
Thank you for the explanation. Resolving.